### DISCUSSION TITLE
Can't locate friend despite knowing their Mastodon username

### DISCUSSION THREAD

**Discussion by DrBlackberry** [mastodon/mastodon#21057] [Q&A] [answered]
I have a friend who joined Mastodon yesterday and I know their username. I have tried on a number of occasions to search for them but I keep getting no results. 

Would greatly appreciate some friendly advice.

**Comment by DrBlackberry** [comment, marked as answer]
Solved by putting in the full address ie The username + the server.   However there are a number of users that can be found by just putting in the username only. Not sure why this is the case

**Reply by trwnh** [reply]
Usernames work if the user is local. If the user is not local, then you must include the domain.
